### DISCUSSION TITLE
Pin dependencies not taking pnpm-lock.yaml into consideration

### DISCUSSION THREAD

**Discussion by pdcmoreira** [renovatebot/renovate#19016] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Basically https://github.com/renovatebot/renovate/issues/1362 should also work for `pnpm-lock.yaml` the same way it does for `package-lock.json`, otherwise the "Pin dependencies" PR that is created is trying to update all packages to the latest versions while pinning.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment, marked as answer]
not a bug. please resubmit as feature request instead.

**Reply by pdcmoreira** [reply]
Sorry, I assumed pnpm was supposed to be fully supported (as in having feature parity).

**Reply by viceice** [reply]
it is fully supported, but sometimes features aren't in parity 🤷‍♂️

**Reply by rarkins** [reply]
Please submit a feature request for "Support version pinning for pnpm" and include a reproduction repo which can be tested against

**Reply by pdcmoreira** [reply]
Created the feature request: https://github.com/renovatebot/renovate/issues/19038
